tgui/packages/tgui/interfaces/PlayerPreferences.tsx

```diff
@@ -0,0 +1,1502 @@
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={employmentDesc}
+              onChange={(e, value) => setEmploymentDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+      </Grid>
+      <Grid>
+        <Grid.Column>
+          <Section
+            title="Security Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={securityDesc === sec_record}
+                  onClick={() => act('sec_record', { securityDesc })}>
+                  Save
+                </Button>
+                <Button
+                  icon="times"
+                  onClick={() => setSecurityDesc(sec_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={securityDesc}
+              onChange={(e, value) => setSecurityDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+        <Grid.Column>
+          <Section
+            title="Exploit Records"
+            buttons={
+              <Box>
+                <Button
+                  icon="save"
+                  disabled={exploitsDesc === exploit_record}
+                  onClick={() => act('exploit_record', { exploitsDesc })}>
+                  Save
+                </Button>
+                <Button
+                  icon="times"
+                  onClick={() => setExploitsDesc(exploit_record)}>
+                  Reset
+                </Button>
+              </Box>
+            }>
+            <TextArea
+              height="100px"
+              maxLength={1024}
+              value={exploitsDesc}
+              onChange={(e, value) => setExploitsDesc(value)}
+            />
+          </Section>
+        </Grid.Column>
+      </Grid>
+    </Section>
+  );
+};
+
+const CharacterCustomization = (props, context) => {
+  const { act, data, config } = useBackend<PlayerPreferencesData>(context);
+  const {
+    random_name,
+    r_hair,
+    g_hair,
+    b_hair,
+    r_grad,
+    g_grad,
+    b_grad,
+    r_eyes,
+    g_eyes,
+    b_eyes,
+    r_facial,
+    g_facial,
+    b_facial,
+  } = data;
+
+  const rgbToHex = (red, green, blue) => {
+    const convert = comp => {
+      const hex = comp.toString(16);
+      return hex.length === 1 ? `0${hex}` : hex;
+    };
+    return '#' + convert(red) + convert(green) + convert(blue);
+  };
+
+  return (
+    <>
+      <Section title="Profile">
+        <Flex>
+          <Flex.Item>
+            <LabeledList>
+              <TextFieldPreference
+                label={'Full Name'}
+                action={'name_real'}
+                value={'real_name'}
+                extra={
+                  <span>
```

Comment:
what does this span do, use <Box as="span">
```suggestion
                  <Box as="span">
```